### PR TITLE
Closes #2020: Make sphinx documentation wcag complaint

### LINKED ISSUE #2020: Make sphinx documentation wcag complaint

### PR DESCRIPTION
**Pull request by madryk**
Things that was done as part of this task:
 - added `skip-to-content` link (https://www.w3.org/TR/WCAG20-TECHS/G1.html)
 - bumped some of the contrasts (mainly: links in the left side menu, code highlights, quite a lot of things that was used in `patterns.rst`
 - removed links to google analytics
 - now all of the images are figures that can be references from the text (they are auto numbered). Added captions for all of them. If some image had a text that was easy transferred to a caption (cropping) - it was done.
 - wherever I found, stumped on example of block code written by \`\`command execute something\`\` I replaced it either with `.. code-block::` directive or used `::` shortcut) - it will be rendered differently: first renders red text and the second render grey box
 - fixed some ordered and unordered lists (some of examples was breaking current lists and starting new ones for each point)
 - marked first row as headers in some tables
 - fixed anchor links
 - removed / replaced ocurrences of above/below. I was either replace them completely or change to some "synonyms" that points more accurate where is something that is described, if for example i found statement that something is below and this "below" meant couple of headers below then I tried to add some reference link. (I didn't check / process user guide in this way - this will be done by @wfenrich )
 - some of the links with text `here` was replaced with more meaningful text
 - in left menu search box now how button for submitting searched phrase
 - corrected some of the invalid links that pointed to non existing pages in documentation (`dev-installation` - if I remember correctly)

### CODE REVIEW

File: doc/sphinx-guides/source/admin/metadatacustomization.rst [hunk 1]
```diff
@@ -616,7 +624,7 @@ Footnotes
    https://www.iana.org/assignments/media-types/text/tab-separated-values
 
 .. [2]
-   Although the structure of the data, as you’ll see below, violates the
+   Although the structure of the data, can violate the
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/admin/metadatacustomization.rst]: Unnecessary comma.

File: doc/sphinx-guides/source/api/native-api.rst [hunk 2]
```diff
@@ -703,7 +711,7 @@ For example::
 
 If the dataset is not locked (or if there is no lock of the specified type), the API will exit with a warning message. 
 
-(Note that the API calls above all support both the database id and persistent identifier notation for referencing the dataset)
+(Note that all of this API calls support both the database id and persistent identifier notation for referencing the dataset)
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/api/native-api.rst]: these

File: doc/sphinx-guides/source/api/native-api.rst [hunk 3]
```diff
@@ -1380,7 +1388,7 @@ Execute a saved search by database id and make links to dataverses and datasets
 
   PUT http://$SERVER/api/admin/savedsearches/makelinks/$id?debug=true
 
-Execute all saved searches and make links to dataverses and datasets that are found. ``debug`` works as described above.  ::
+Execute all saved searches and make links to dataverses and datasets that are found. ``debug`` parameters works in the same manner as in making links for single saved search::
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/api/native-api.rst]: parameter?

File: doc/sphinx-guides/source/api/search.rst [hunk 4]
```diff
@@ -227,7 +227,7 @@ In this example, ``show_relevance=true`` matches per field are shown. Available
 Iteration
 ---------
 
-Be default, up to 10 results are returned with every request (though this can be increased with the ``per_page`` parameter). To iterate through many results, increase the ``start`` parameter on each iteration until you reach the ``total_count`` in the response. An example in Python is below.
+Be default, up to 10 results are returned with every request (though this can be increased with the ``per_page`` parameter). To iterate through many results, increase the ``start`` parameter on each iteration until you reach the ``total_count`` in the response. An example of such iteration Python:
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/api/search.rst]: in Python

File: doc/sphinx-guides/source/developers/containers.rst [hunk 5]
```diff
@@ -138,17 +150,17 @@ If you're interested in using Minishift for development and want to change the D
...
+First, set the Docker environment variables so that ``docker build`` and ``docker images`` refer to the internal Minishift registry rather than your normal Docker setup::
 
-``eval $(minishift docker-env)``
+    eval $(minishift docker-env)
 
-When you're ready to build, change to the right directory:
+When you're ready to build, change to the right directory::
 
-``cd conf/docker``
+    cd conf/dockermethod
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/developers/containers.rst]: Is the ``dockermethod`` correct?

File: doc/sphinx-guides/source/developers/dependencies.rst [hunk 6]
```diff
@@ -140,7 +150,7 @@ Maven can manage versions of transitive dependencies in four ways:
    and add a ``<version>`` tag. Many bigger and standard use projects provide those, making the POM much less bloated
    compared to adding every bit yourself.
 
-A reduced example, only showing bits relevant to the above cases and usage of an explicit transitive dep directly:
+A reduced example, only showing bits relevant to this cases and usage of an explicit transitive dep directly:
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/developers/dependencies.rst]: these

File: doc/sphinx-guides/source/developers/selinux.rst [hunk 7]
```diff
@@ -20,7 +18,7 @@ This document is something of a survival guide for anyone who is tasked with upd
 Development Environment
 -----------------------
 
-In order to work on the ``shibboleth.te`` file you need to ``ssh`` into a RHEL or CentOS box running Shibboleth (instructions are in the :doc:`/installation/shibboleth` section of the Installation Guide) such as https://beta.dataverse.org or https://demo.dataverse.org that has all the commands below installed. As of this writing, the ``policycoreutils-python`` RPM was required.
+In order to work on the ``shibboleth.te`` file you need to ``ssh`` into a RHEL or CentOS box running Shibboleth (instructions are in the :doc:`/installation/shibboleth` section of the Installation Guide) such as https://beta.dataverse.org or https://demo.dataverse.org that has installed all the mention commands in this document. As of this writing, the ``policycoreutils-python`` RPM was required.
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/developers/selinux.rst]: ``all the commands mentioned in this document``?

File: doc/sphinx-guides/source/developers/tools.rst [hunk 8]
```diff
@@ -16,7 +14,7 @@ http://wiki.netbeans.org/ChromeExtensionInstallation
 pgAdmin
 +++++++
 
-You probably installed pgAdmin when following the steps in the :doc:`dev-environment` section but if not, you can download it from https://www.pgadmin.org
+pgAdmin is an graphical tool to manage your database. You can download it from https://www.pgadmin.org
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/developers/tools.rst]: `a graphical`

File: doc/sphinx-guides/source/developers/version-control.rst [hunk 9]
```diff
@@ -53,7 +53,7 @@ Pull requests take all shapes and sizes, from a one-character typo fix to hundre
 
 If you are writing code (rather than documentation), please see :doc:`testing` for guidance on writing tests.
 
-The example of creating a pull request below has to do with fixing an important issue with the :doc:`documentation` but applies to fixing code as well.
+The example of creating a pull request presented in this section and it's subsections has to do with fixing an important issue with the :doc:`documentation` but applies to fixing code as well.
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/developers/version-control.rst]: `its`

File: doc/sphinx-guides/source/installation/config.rst [hunk 10]
```diff
@@ -658,7 +676,7 @@ DoiBaseUrlString
 
 As of this writing, "https://mds.datacite.org" (DataCite) and "https://ezid.cdlib.org" (EZID) are the main valid values.
 
-While the above two options are recommended because they have been tested by the Dataverse team, it is also possible to use a DataCite Client API as a proxy to DataCite. In this case, requests made to the Client API are captured and passed on to DataCite for processing. The application will interact with the DataCite Client API exactly as if it were interacting directly with the DataCite API, with the only difference being the change to the base endpoint URL.
+While this two options are recommended because they have been tested by the Dataverse team, it is also possible to use a DataCite Client API as a proxy to DataCite. In this case, requests made to the Client API are captured and passed on to DataCite for processing. The application will interact with the DataCite Client API exactly as if it were interacting directly with the DataCite API, with the only difference being the change to the base endpoint URL.
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/installation/config.rst]: `these`

File: doc/sphinx-guides/source/installation/prerequisites.rst [hunk 11]
```diff
@@ -52,35 +52,35 @@ Glassfish Version 4.1.2 is required.
 Installing Glassfish
 ====================
 
-**Note:** The Dataverse installer need not be run as root, and it is recommended that Glassfish not run as root either. We suggest the creation of a glassfish service account for this purpose.
+.. note:: The Dataverse installer need not be run as root, and it is recommended that Glassfish not run as root either. We suggest the creation of a glassfish service account for this purpose.
 
-- Download and install Glassfish (installed in ``/usr/local/glassfish4.1.2`` in the example commands below)::
+Examples in this section assumes that you want to install glassfish in ``/usr/local/glassfish4.1.2`` directory.
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/installation/prerequisites.rst]: `assume`

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 12]
```diff
@@ -497,10 +552,10 @@ skills.
 III. What the install.pl script does:
 -------------------------------------
 
-The steps below are performed by the ``install.pl`` script. **Provided for reference only!** 
-The instruction below could be used to configure it all by hand, if necessary, or 
+The instruction presented in this section are performed by the ``install.pl`` script. **Provided for reference only!** 
+It could be used to configure it all by hand, if necessary, or 
 to verify that the installer has done it correctly. 
-Once again: **normally you would NOT need to individually perform the steps below**!
+Once again: **normally you would NOT need to individually perform the this steps**!
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst]: ~~the~~ `these steps`

File: doc/sphinx-guides/source/user/dataset-management.rst [hunk 13]
```diff
@@ -387,7 +402,7 @@ In the Widgets tab, you can copy and paste the code snippets for the widget you
 Dataset Widget
 ~~~~~~~~~~~~~~
 
-The Dataset Widget allows the citation, metadata, files and terms of your dataset to be displayed on your website. When someone downloads a data file in the widget, it will download directly from the datasets on your website. If a file is restricted, they will be directed to your dataverse to log in, instead of logging in through the widget on your site. 
+The Dataset Widget allows the citation, mhttps://jasnopis.pl/ - fajna apkaetadata, files and terms of your dataset to be displayed on your website. When someone downloads a data file in the widget, it will download directly from the datasets on your website. If a file is restricted, they will be directed to your dataverse to log in, instead of logging in through the widget on your site.
```
Comment (wtr-icm) [inline doc/sphinx-guides/source/user/dataset-management.rst]: ?

Comment (madryk) [reply doc/sphinx-guides/source/user/dataset-management.rst]: It looks like I did ctrl+v by mistake. I will revert that like it was before

File: doc/sphinx-guides/source/developers/containers.rst (hunk 5 above)
Comment (madryk) [reply doc/sphinx-guides/source/developers/containers.rst]: No. It is a mistake

**Review by wtr-icm — APPROVED** [review]

**Pull request merged**